td-agent/Rakefile

```diff
@@ -246,9 +246,20 @@ class BuildTask
 
   def make_wix_version_number(version)
     return version unless version.include?("~")
+    revision = ""
+    case version
+    when /~rc(\d+)/
+      revision = $1.to_i * 10000 + Time.now.hour
+    when /~beta(\d+)/
+      revision = $1.to_i * 1000 + Time.now.hour
+    when /~alpha(\d+)/
+      revision = $1.to_i * 100 + Time.now.hour
```

Comment:
revison should be from 0 to 65535.
Should we check version number overflow here?

Reply:
I got it. I'll fix it.

Reply:
pushed commit to fix it.